Gemfile

```diff
@@ -27,6 +27,6 @@ else
   gem 'puppet', :require => false
 end
 
-gem 'hocon', '~> 0.9.3',       :require => false
```

Comment:
/cc @DavidS for thoughts on this. We're currently working on adding hocon to modulesync and I'm not sure if we have any policies/best practices around this.

Thanks for bringing it up @cprice404!